### ISSUE TITLE
Python: CreateRenderTask with page size options does not produce same output as manual download

### ISSUE THREAD

**Issue by ContrastingSounds** [looker-open-source/sdk-codegen#167]
If I make an sdk.create_dashboard_render_task() call with the same settings as the manual download (e.g. width 1680, landscape, page_size A4) I don't get the same PDF as a result. Similarly for A3. Have only tested with the Python SDK.

As well as not being the same output as manual download (which is what I would consider the expected behaviour), it doesn't produce an A4 or A3 page.

This is critical for an upcoming trial – also, I've seen a LOT of commentary at different accounts trying to get good, predictable PDF output. I'd imagine there's also a rendering aspect to this, but the primary requirement I believe is being able to use the API to get the same output as achievable via the UI.

Related nice to haves: 
- drop height as a mandatory param as it doesn't seem to be used.  
- if given a page_size, don't require a width
- make it possible to specify both width and page_size independently (because tables in particular will render differently for the same page size, if given different widths

**Comment by jkaster** [comment]
I don't think this is a python issue. Sounds like it's more of a Looker API endpoint issue, and the bug should be entered against Looker rather than the SDK
